drivers/char/rust_example.rs

```diff
@@ -52,6 +52,13 @@ impl KernelModule for RustExample {
         println!("  my_bool:  {}", my_bool.read());
         println!("  my_i32:   {}", my_i32.read());
 
+        // Including this large variable on the stack will trigger a call to
+        // `compiler_builtins::probestack::__rust_probestack` on x86_64.
+        // This will verify that we are able to link modules which call
+        // `__rust_probestack`.
+        let x: [u64; 1028] = [5; 1028];
```

Comment:
```suggestion
        let x: [u64; 1028] = core::hint::black_box([5; 1028]);
```

Requires `#![feature(test)]`, but ensures that it isn't optimized out.